californium-core/src/main/java/org/eclipse/californium/core/network/CoapEndpoint.java

```diff
@@ -1330,6 +1342,17 @@ public Builder setCoapStackFactory(CoapStackFactory coapStackFactory) {
 			return this;
 		}
 
+		/**
+		 * Set logging tag.
+		 * 
+		 * @param tag logging tag. Defautls to connector's scheme.
+		 * @return this
+		 */
+		public Builder setLoggingTag(String tag) {
+			this.tag = tag;
```

Comment:
I'm not sure but should we normalize tag here ?

Reply:
Without there, `null`could be applied to use the default tag.